Misc/voting on deciding the technology stack

```diff
@@ -32,30 +36,10 @@ DB2     ---
 (we need to figure out a way to setup rds for this db so that when we deploy to aws or other server we should link it)
 
 Messaging Queues: (your name -- reason)
-Active MQ  ---
+Active MQ  ---(PRANAV RAYAPROLU) -- active MQ I worked on and Infact am okay for any queue. 
 rabbit MQ --- (Manoj chitabathina -- also this because i know how to use this. simple and quick and very reliable)
 kafka    ----(Manoj chitabathina -- beacuse i dont know about this and this is something i wanted to know)
 IBM MQ ---
 
-Security: (your name -- reason)
-Spring Security:(Manoj chitabathina -- because we can have the control over it and its free and  i dont know whether others are free or not )
-some other 3rd party Login apis (ex: google, aws cognito, Okta.. etc)
```

Comment:
please make sure you are not removing any lines in this document @rayapr please add all the lines back